### PR TITLE
RD-2000 Enable consistent-type-imports eslint rule

### PR DESCRIPTION
**Pull request by kubama**
## Description
Enabled consistent-type-imports eslint rule by updating cloudify-ui-common version.

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
N/A

 

## Documentation
N/A

### CODE REVIEW

File: app/reducers/pageReducer.ts [hunk 1]
```diff
@@ -6,19 +6,10 @@ import type { AnyAction, Reducer } from 'redux';
...
-    isWidgetsSection,
-    LayoutSection,
-    moveTab,
-    PageDefinition,
-    removeTab,
-    TabContent,
-    updateTab
-} from '../actions/page';
-import { createPagesMap, InsertPosition, PageGroup, PageMenuItem } from '../actions/pageMenu';
+import type { addTab, LayoutSection, moveTab, PageDefinition, removeTab, TabContent, updateTab } from '../actions/page';
```
Comment (qooban) [inline app/reducers/pageReducer.ts:9]: This is interesting. Seems like functions returning object `as const` (e.g. `addTab`, `moveTab`) are treated as types.

File: app/reducers/managerReducer/licenseReducer.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import type { Reducer } from 'redux';
 import * as types from '../../actions/types';
 import Auth from '../../utils/auth';
-import Consts from '../../utils/consts';
+import type Consts from '../../utils/consts';
```
Comment (qooban) [inline app/reducers/managerReducer/licenseReducer.ts:4]: This is interesting. Seems like auto-fixer is quite smart to detect in which context (JS or TS) imported file is used and then decides if to use `import` or `import type`.

**Review by qooban — APPROVED** [review]
Looks good! 

I'm assuming that all changes were made with auto-fixer. That's great we won't need to fix this kind of issues manually anymore 👍🏻

File: app/reducers/pageReducer.ts (hunk 1 above)
Comment (kubama) [reply app/reducers/pageReducer.ts:9]: The thing here is that those functions are used only as `typeof` operands.

File: app/reducers/managerReducer/licenseReducer.ts (hunk 2 above)
Comment (kubama) [reply app/reducers/managerReducer/licenseReducer.ts:4]: Similarly here, it is used only as `typeof` operand.

File: app/reducers/pageReducer.ts (hunk 1 above)
Comment (qooban) [reply app/reducers/pageReducer.ts:9]: 👍🏻

File: app/reducers/managerReducer/licenseReducer.ts (hunk 2 above)
Comment (qooban) [reply app/reducers/managerReducer/licenseReducer.ts:4]: 👍🏻

**Pull request merged**